pkg/minishift/registration/utils.go

```diff
@@ -35,21 +34,24 @@ type RegistrationParameters struct {
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 	commander := provision.GenericSSHCommander{Driver: host.Driver}
-	registrator, err := DetectRegistrator(commander)
-	if err == ErrDetectionFailed {
+	registrator, supportRegistration, err := DetectRegistrator(commander)
+	if ! supportRegistration {
```

Comment:
Same as comment below. I don't think we should log this on info level. If anything debug level.